rasa/constants.py

```diff
@@ -38,3 +38,5 @@
 ENV_GPU_CONFIG = "TF_GPU_MEMORY_ALLOC"
 ENV_CPU_INTER_OP_CONFIG = "TF_INTER_OP_PARALLELISM_THREADS"
 ENV_CPU_INTRA_OP_CONFIG = "TF_INTRA_OP_PARALLELISM_THREADS"
+
+EPOCH_OVERRIDE = "epoch_override"
```

Comment:
mhm. I didn't see it as specifically related to `TensorFlow`. Any component which has epochs might use it, right? As this is a general flag for finetuning I think it should also be a general constant. The tensorflow constants seem to specific for me in this case.

Reply:
> Any component which has epochs might use it, right?
I like that idea but we only use it for the tf models (ehm, and only it TED it seems 😅 ) (and although I like that idea, unifying the settings might also not be ideal because sklearn users would expect that to be called "iter")

Reply:
Mhm 🤔 But even if sklearn calls it differently we wouldn't expose this implementation detail to our users / outer layers, right? Will move it over though 👍🏻

Reply:
actually I think you're right. good idea to move it 👍🏻